internal/providers/pke/pkeworkflow/create_cluster.go

```diff
@@ -26,9 +28,24 @@ import (
 const CreateClusterWorkflowName = "pke-create-cluster"
 const pkeVersion = "0.4.8"
 
-func getDefaultImageID(region, kubernetesVersion string) string {
-	switch kubernetesVersion {
-	case "1.12.2":
+func getDefaultImageID(region, kubernetesVersion string) (string, error) {
```

Comment:
All the possible errors are programming errors (occur when the hardcoded constraints are invalid). i would prefer to use `MustNewConstraint` (like regexp.MustCompile), but unfortunately it doesn't exist in the lib. That would simplify the code to something like `switch { case check(kubeVersion, "~1.12.0"):`